src/WebJobs.Extensions.SendGrid/Client/ISendGridClient.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+namespace Client
```

Comment:
I changed this namespace to avoid conflicts we were having with their SendGrid namespace and their SendGridClient. .NET core doesn't support extern alias, which would have been another way to resolve.